packages/adapter-components/src/references/context.ts

```diff
@@ -88,7 +88,12 @@ export const neighborContextGetter = ({
   }
 
   try {
-    const contextPath = getParent(fieldPath, levelsUp).createNestedID(contextFieldName)
+    const parent = getParent(fieldPath, levelsUp)
+    if (parent.isConfig()) {
```

Comment:
why not check in advance (before trying to resolve `getParent`)?
```suggestion
    if (levelsUp >= fieldPath.nestingLevel) {
       // trying to go up too many levels...
```
(not sure if it should be `>=` or `>`)

Reply:
`nestingLevel ` is not the same, because we ignore arrays when going up. I could check in advance but not sure it's really worth it

Reply:
I think it was originally for salesforce where arrays may appear / disappear in the values